### PR TITLE
Add "AiTutorInteractionFeedbacks" to daily DMS task

### PR DESCRIPTION
**Pull request by coryamanda**
I have not done this before so PLEASE confirm that it is right so I don't break anything. The goal here is to add the "AiTutorInteractionFeedbacks" table to the daily DMS task so we can expose it to Product. I don't know if:
- this is the exact name of the table (naming conventions are different than rest of file)
- it's in dashboard or dashboard_pii

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: aws/dms/tasks.yml [hunk 1]
```diff
@@ -51,6 +51,7 @@ cron-pii:
 - dashboard.learning_goal_ai_evaluations
 - dashboard.learning_goal_teacher_evaluations
 - dashboard.learning_goal_ai_evaluation_feedbacks
+- dashboard.aitutorinteraction_feedbacks
```
Comment (cat5inthecradle) [inline aws/dms/tasks.yml]: This table is actually called `ai_tutor_interaction_feedbacks`. There are a couple of ways to discover that. The [ActiveRecord model](https://github.com/code-dot-org/code-dot-org/blob/12814a8249fec8c9589c3ed28a47a97c392f0d8d/dashboard/app/models/ai_tutor_interaction_feedback.rb) lists it's table name, for example. You could also check [schema.rb](https://github.com/code-dot-org/code-dot-org/blob/10adb73681521aac29d80e4f3c5e039fafb1d573/dashboard/db/schema.rb). I ran `select tables;` on the database to check. 

```suggestion
- dashboard.ai_tutor_interaction_feedbacks
```

**Review by cat5inthecradle — CHANGES_REQUESTED** [review]
The table name is incorrect, that's the only mandatory change I see.

As for PII, you've placed this in the pii schema, which might be correct .

The following columns exist in the table. I'm not certain what the `details` column contains, but it seems possible that it could contain PII as it's user-input. If so, then putting it under `cron-pii` is correct.

```
id
ai_tutor_interaction_id
user_id
thumbs_up
thumbs_down
created_at
updated_at
details
```

**Comment by cat5inthecradle** [comment]
@ebeastlake You added this model to the codebase, can you help us understand whether this should be considered PII? If so, we'll send it to the pii schema in Redshift - or we could remove the pii or anonymize it before sending to the non-pii schema.

**Comment by coryamanda** [comment]
fixed the table name. I assumed that this could potentially contain PII and so added it there, but even better if @ebeastlake can confirm

**Comment by ebeastlake** [comment]
Whoops, sorry for the missed pings! I wouldn't expect AI Tutor interaction feedback ("Why did you flag this response?") contains PII, but it theoretically could. I was looking to see where the regular AI Tutor interactions table was for comparison/verification, but I don't see it listed in `aws/dms/tasks.yml`. Should I be looking somewhere else?

**Comment by ebeastlake** [comment]
`ai_tutor_interactions` did not go in the `pii_` stack, which might have been a miss. 
Screenshot 2024-08-28 at 1 38 18 PM

**Comment by coryamanda** [comment]
I just looked at the AI Tutor Interactions table in Trevor and don't see anything there that looks like PII. I moved over the feedback table to the non-PII schema unless anyone objects.

**Review by cat5inthecradle — APPROVED** [review]

**Comment by cat5inthecradle** [comment]
This column:
- Is not intended to store PII
- Does not appear to have been used to store PII inadvertently

Approving, will deploy today.

**Comment by cat5inthecradle** [comment]
Deployed.

```
❯ bundle exec rake stack:data:validate RAILS_ENV=production ADMIN=true
...
Finished stack:data:environment (less than a minute)
Pending update for stack `DATA-production`:
Modify DMSCron [AWS::DMS::ReplicationTask] Properties Replacement: Conditional (TableMappings)
Finished stack:data:validate (less than a minute)
❯ bundle exec rake stack:data:start RAILS_ENV=production ADMIN=true
...
Finished stack:data:environment (less than a minute)
Pending update for stack `DATA-production`:
Modify DMSCron [AWS::DMS::ReplicationTask] Properties Replacement: Conditional (TableMappings)
Proceed? [y/n]
y
 Stack update requested, waiting for provisioning to complete...
.2024-09-03 21:52:02 UTC- DMSCron [UPDATE_IN_PROGRESS]
.......2024-09-03 21:52:48 UTC- DMSCron [UPDATE_COMPLETE]
2024-09-03 21:52:50 UTC- DATA-production [UPDATE_COMPLETE_CLEANUP_IN_PROGRESS]
2024-09-03 21:52:48 UTC- DMSCron [UPDATE_COMPLETE]
2024-09-03 21:52:50 UTC- DATA-production [UPDATE_COMPLETE_CLEANUP_IN_PROGRESS]

Stack update complete.
Outputs:
Finished stack:data:start (3 minutes)
```

**Pull request merged**